packages/ember-runtime/lib/system/native_array.js

```diff
@@ -138,9 +138,10 @@ var A;
 
 if (Ember.EXTEND_PROTOTYPES === true || Ember.EXTEND_PROTOTYPES.Array) {
   NativeArray.apply(Array.prototype);
-  A = function (arr = []) { return arr; };
+  A = function (arr = []) { return arr || []; };
 } else {
   A = function (arr = []) {
+    if (arr === null) { arr = []; }
```

Comment:
:sob: truthiness was the mode with prototype extensions enabled (which is the most common), so I agree normalizing around that as a BUGFIX makes sense :+1: